packages/rollup-plugin-styles/src/loaders/sass/index.ts

```
@@ -0,0 +1,82 @@
+        const render = async (options: sass.Options): Promise<sass.Result> =>
+            // eslint-disable-next-line compat/compat
+            await new Promise((resolve, reject) => {
+                if (sync) {
+                    resolve(sass.renderSync(options));
+                } else {
+                    sass.render(options, (error, css) => (error ? reject(error) : resolve(css)));
+                }
+            });
+
+        // Remove non-Sass options
+        delete options.impl;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)